src/MovieCard/MovieCard.test.js

```diff
@@ -1,14 +1,46 @@
-import { render, screen, fireEvent } from '@testing-library/react';
-import '@testing-library/jest-dom'
+import React from 'react';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import userEvent from '@testing-library/user-event';
 import MovieCard from './MovieCard';
+import { getAllMoviesData, getSingleMovieData } from '../apiCalls.js'
```

Comment:
We can remove this import since it is already passed into movie card through props. :)